packages/fe-fpm-writer/src/building-block/index.ts

```diff
@@ -156,25 +172,30 @@ function getMetaPath(
  *
  * @param {BuildingBlock} buildingBlockData - the building block data
  * @param {Document} viewDocument - the view xml file document
+ * @param {Manifest} manifest - the manifest content
  * @param {Editor} fs - the memfs editor instance
  * @param {boolean} usePlaceholders - apply placeholder values if value for attribute/property is not provided
  * @returns {string} the template xml file content
  */
 function getTemplateContent<T extends BuildingBlock>(
     buildingBlockData: T,
     viewDocument: Document | undefined,
+    manifest: Manifest | undefined,
     fs: Editor,
     usePlaceholders?: boolean
 ): string {
     const templateFolderName = buildingBlockData.buildingBlockType;
     const templateFilePath = getTemplatePath(`/building-block/${templateFolderName}/View.xml`);
     if (typeof buildingBlockData.metaPath === 'object' || buildingBlockData.metaPath === undefined) {
+        // Special handling for chart - while runtime does not support approach without contextPath
+        // or for equal or below UI5 v1.96.0 contextPath is applied
+        const minUI5Version = manifest ? coerce(getMinimumUI5Version(manifest)) : undefined;
+        const applyContextPath =
+            buildingBlockData.buildingBlockType === BuildingBlockType.Chart ||
+            !minUI5Version ||
```

Comment:
I think if `minUI5Version` is not resolved we consider it as latest, example of existing check -> https://github.com/SAP/open-ux-tools/blob/main/packages/fe-fpm-writer/src/common/validate.ts#L16

Reply:
Fixed: if not defined, contextPath is NOT applied to the building block